### ISSUE TITLE
Can not find a way to quickly select another clip from media window into preview via keyboard

### ISSUE THREAD

**Issue by softwarelimits** [jliljebl/flowblade#624]
This is a keyboard usability and workflow issue. I am doing this - I am not sure, if this is the workflow you want users to follow, however it is a workflow I am used to apply using several different video editing softwares since many years and I believe this is a quite usual way of doing things. In principle Flowblade allows me to do that, but it does not allow me to do that _quickly and only with the keyboard_. 

I load up a pile of media clips from some folder. Then I want to preview one clip after the other in the monitor window and set IN and OUT markers, optionally sort clips into dedicated bins - this process is annoying and tiresome and the editing software should support the user to do this as quickly as possible and the software absolutely must avoid to disrupt or annoy the user doing this work.

How I said this works, but it does not work without using the mouse, also I noticed some focus irritations with the keyboard, e.g. many times the HOME and END buttons did not work as expected after a double click in the media window, which has to happen because I need to start the preview of a clip with a mouseclick and found no way to switch to the media window with the keyboard.

I try to explain what is needed - I think it would absolutely make sense if you just tried to do that _with the keyboard only_ then you will better understand, what I am trying to describe. Imagine coming back to the studio with A LOT of shots. You put all the clips into one folder and open it into a media window in Flowblade. OK, now we need to get all the usable / good / not too bad sequences from that pile of clips - this is _not the creative part of the work, it is just an annoying step that is neccessary before we can start with the creative work_ - and this is exactly the reason why this work has to be supported by the software to be done as quickly as possible. 

So I want to: 
* use one keyboard shortcut to select the next clip into the preview
* play the clip (in LOOP mode, see my other issue) and select IN and OUT points - that seems to work well.
* optionally categorize this clip by putting it quickly into some bin - with one keyboard shortcut
* optionally add the selected IN-OUT range to the timeline, with one keyboard shortcut, what seems to work well
* but then **put the next clip into the preview monitor with a keyboard shortcut** - no way to do that with Flowblade, at this point I have to reach out to the mouse - annoying! Instead I just want the next clip to be put on the monitor without leaving the keyboard, because this must happen _quickly_.
* repeat.

I assume that you have seen how people work with software that they are using every day for many hours since many years? I mean have you seen that they work _really quick_ and mostly they _have to because every minute counts_. THAT kind of approach to support _quick_ user workflows is missing from Flowblade, and this is the key element that allows the use of the word "professional" in any software description - simply _allow the user to perform the neccessary steps with the least possible interactions_, so they can hit the deadline.

Please do not misunderstand me - I have high hopes for Flowblade! Version 2 looks really good and I am playing the whole afternoon with this new release and have had no single crash - what is really a very positive experience compared to other tools in this area. _Because I like very much what you did_ I allow myself to post these issues. 

As you seem to have some real experience from a professional video work background I hope you will understand that just a few tweaks regarding the GUI / usability and keyboard shortcuts can push Flowblade into a real professional workhorse, where these kind of micro-tunings for _very quick usage_ are very important to get right - as you know most of video editing jobs have extremely restricted budget and it will be an incredible help for pressured editors if Flowblade supported these kind of micro-patterns for very quick keyboard-based workflows that can help save a lot of time for the real work to be done. 

Thank you very much for your attention and your incredible work!

**Comment by softwarelimits** [comment]
BTW this can and should be expanded to the timeline - so 
* switching from timeline to media clips
* generating a range 
* inserting range into timeline
* move around the timeline
* switch back to media clips
* select range + add to timeline
* repeat

should all work _without having to touch the mouse_ - I can not find a way to do that without mouse hitting, what destroys a _quick workflow_ for editing sessions. 

Generally speaking, it would be really fantastic if you tried to use Flowblade with keyboard only - it looks like you are really just a few tweaks away from this ultra-professional level - please add this killer features and flowblade will blow away all the other tools! From what I understand this is just a little bit of tuning the behaviour of the GUI keyboard and focus handling, I guess this is less than one day of work for you - the result will be an incredible heavy industry editing tool with high speed keyboard only workflow - I know a few people that will get crazy about that :) ... please keep up the good work!

**Comment by jliljebl** [comment]
I get the use case you are looking for: being able to loop through a lot of clips and adding them into timeline as quickly as possible...and never touching the mouse. Using only mouse is indeed always faster if possible.

We will be adding new keyboard shortcuts this cycle and being able to do the workflow you describe can be one of the goals as we go along.

**Comment by jliljebl** [comment]
I added keyboard shortcut "ALT + M" that opens in and selects *next* media item after currently selected media item. It also makes monitor grab keyboard focus which allows for hitting space to play without any mouse actions.

This is about as much as we can do for this release,  open new issue if you have further ideas on this area.

Feature will be in 2.2.
